### PR TITLE
96 modify candidate view questions screen

### PR DESCRIPTION
**Pull request by Dhruv-Copods**
Following UI Changes Implemented,

- Refactored Candidate Questions Page UI

### CODE REVIEW

File: app/components/assessment/CandidateQuestionFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  candidateExamConstants,
+  commonConstants,
+} from '~/constants/common.constants'
+import Button from '../form/Button'
+import CandidateQuestionStepper from './CandidateQuestionStepper'
+
+const CandidateQuestionFooter = () => {
+  const { question, section, lastSection } = useLoaderData()
+  return (
+    <div className="flex h-16 items-center gap-6 rounded-b-lg border-t bg-white px-6 py-4 drop-shadow-[0px_-2px_10px_rgba(0,0,0,0.06)]">
```
Comment (yash6102000) [inline app/components/assessment/CandidateQuestionFooter.tsx:12]: please if if not available in tailwind class////then add in tailwind config

File: app/components/assessment/Question.tsx [hunk 2]
```diff
@@ -30,138 +44,153 @@ const Question = () => {
...
+    //   old[index as number].isCorrect = !event.isCorrect
+    //   console.log('asd', ...old)
+    //   return [...old]
+    // })
+    // question?.question?.options.forEach((opt: any) => {
+    //   if (event.id === opt.id) {
+    //     opt.isCorrect = !opt.isCorrect
+    //     console.log(question.question.options)
+    //     // let something = [...question.question.options]
+    //     // console.log(something)
```
Comment (yash6102000) [inline app/components/assessment/Question.tsx:68]: if we are not going to use this commented code further then ....please remove it

File: app/models/assessment.server.ts [hunk 3]
```diff
@@ -367,6 +368,7 @@ export async function skipAnswerAndNextQuestion({
         oldAnswers.push(option.id)
       }
     })
+    console.log('acasc cCQR', currentQuestionId)
```
Comment (yash6102000) [inline app/models/assessment.server.ts:371]: remove the consoles

File: app/models/assessment.server.ts [hunk 4]
```diff
@@ -347,6 +347,7 @@ export async function skipAnswerAndNextQuestion({
   answers: Array<string>
 }) {
   try {
+    console.log('casdf', currentQuestionId)
```
Comment (yash6102000) [inline app/models/assessment.server.ts:350]: remove the consoles

File: tailwind.config.js [hunk 5]
```diff
@@ -9,12 +9,14 @@ module.exports = {
...
         primaryDisabled: '#A2A4D6',
         primaryOutlined: '#141649',
+        questionBackground: '#F0F0F0',
       },
       minWidth: {
         200: '200px',
         260: '260px',
         184: '184px',
         96: '384px',
+        16: '16px',
```
Comment (yash6102000) [inline tailwind.config.js:19]: please use w-4 ..... class provided by tailwind

File: tailwind.config.js [hunk 6]
```diff
@@ -9,12 +9,14 @@ module.exports = {
         primaryHover: '#141649',
         primaryDisabled: '#A2A4D6',
         primaryOutlined: '#141649',
+        questionBackground: '#F0F0F0',
       },
       minWidth: {
         200: '200px',
         260: '260px',
         184: '184px',
         96: '384px',
```
Comment (yash6102000) [inline tailwind.config.js:18]: please use w-96 ..tailwind class

File: tailwind.config.js (hunk 5 above)
Comment (Dhruv-Copods) [reply tailwind.config.js:19]: भाई मिनिमम  विड्थ है वो |

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**